siliconcompiler/tools/klayout/klayout_export.py

```diff
@@ -243,15 +257,18 @@ def read_fills(top):
   in_def = schema.get('input', 'layout', 'def', step=sc_step, index=sc_index)[0]
 else:
   in_def = os.path.join('inputs', f'{design}.def')
-out_gds = os.path.join('outputs', f'{design}.gds')
+out_file = os.path.join('outputs', f'{design}.{sc_stream}')
 
 libs = schema.get('asic', 'logiclib', step=sc_step, index=sc_index)
 if 'macrolib' in schema.getkeys('asic'):
   libs += schema.get('asic', 'macrolib', step=sc_step, index=sc_index)
 
-in_gds = []
+in_files = []
 for lib in libs:
-  in_gds.extend(schema.get('library', lib, 'output', sc_stackup, 'gds', step=sc_step, index=sc_index))
+  for s in sc_streams:
+    if schema.valid('library', lib, 'output', sc_stackup, s):
+      in_files.extend(schema.get('library', lib, 'output', sc_stackup, s, step=sc_step, index=sc_index))
```

Comment:
Same question about duplicate entries in the streams array as above.

Reply:
same, there is a break that would avoid this